ocaml/idl/datamodel_roles.ml

```diff
@@ -41,3 +41,13 @@ let _R_VM_ADMIN = Some(roles_gte role_vm_admin)
 let _R_VM_OP = Some(roles_gte role_vm_operator)
 let _R_READ_ONLY = Some(roles_gte role_read_only) (* = all *)
 let _R_ALL = _R_READ_ONLY
+
+(* Only client-cert; added to one of the above where required in the datamodel *)
+let _R_CLIENT_CERT = Some([role_client_cert])
+
+let union a b =
```

Comment:
I'm not sure what this is meant to do, but this does not look like a union to me. Why is this case not what we want? `Some x, None -> Some x`

Reply:
Oh yes, I forgot to update that.